doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,268 @@
+
+This ``Program`` can be converted into a PennyLane quantum function using the
+:func:`~pennylane.from_pyquil` function:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_pq = qml.from_pyquil(pq_program)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_pq():
+        pl_template_from_pq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The resulting PennyLane circuit is:
+
+>>> print(qml.draw(pl_circuit_from_pq)())
+0: ──H─╭●─┤  <Y>
+1: ────╰X─┤  Var[Z]
+
+.. note::
+
+    Quantum circuits expressed in Quil can be imported in a similar way using
+    :func:`~pennylane.from_quil`.
+
+
+Importing Quantum Operators
+---------------------------
+
+Sometimes, it is preferable to import a single operation from a framework instead of an entire
+quantum circuit. This can save you some keystrokes and serve as a helpful crutch for understanding
+an individual component of a circuit.
+
+Presently, only Qiskit `SparsePauliOp
+<https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp>`__ operators can be
+imported into PennyLane. To see this in action, we first define a ``SparsePauliOp``:
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    qk_op = SparsePauliOp(["II", "XY"])
+
+Then, we apply the :func:`~pennylane.from_qiskit_op` function to convert the ``SparsePauliOp`` into
+a PennyLane :class:`Operator <pennylane.operation.Operator>`:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_op = qml.from_qiskit_op(qk_op)
+
+We can inspect both operators to make sure they match:
+
+>>> qk_op
+SparsePauliOp(['II', 'XY'],
+              coeffs=[1.+0.j, 1.+0.j])
+>>> pl_op
+I(0) + X(1) @ Y(0)
+
+
+Parameterized Operators
+~~~~~~~~~~~~~~~~~~~~~~~
+
+PennyLane also supports importing parameterized ``SparsePauliOp`` instances. Consider:
+
+.. code-block:: python
+
+    import numpy as np
+    from qiskit.circuit import Parameter
+
+    a, b, c = [Parameter(var) for var in "abc"]
+    param_qk_op = SparsePauliOp(["II", "XZ", "YX"], coeffs=np.array([a, b, c]))
+
+To import this ``SparsePauliOp``, we must specify a concrete value for each coefficient using the
+``params`` argument:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    param_pl_op = qml.from_qiskit_op(param_qk_op, params={a: 2, b: 3, c: 4})
+
+The result is:
+
+>>> param_qk_op
+SparsePauliOp(['II', 'XZ', 'YX'],
+              coeffs=[ParameterExpression(1.0*a), ParameterExpression(1.0*b),
+ ParameterExpression(1.0*c)])
+>>> param_pl_op
+(2+0j) * I(0) + (3+0j) * (X(1) @ Z(0)) + (4+0j) * (Y(1) @ X(0))
+
+
+Import Functions
+----------------
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.from_pyquil
+    ~pennylane.from_qasm
+    ~pennylane.from_qasm_file
+    ~pennylane.from_qiskit
+    ~pennylane.from_qiskit_op
+    ~pennylane.from_quil
+    ~pennylane.from_quil_file
+
+:html:`</div>`
```

Comment:
Optional suggestion: might make sense to have this at the top of the page, as the quickstart acts as reference documentation?

Reply:
I'm open to that idea! What do you think, @trbromley?

Reply:
Sounds good, I've left some comments to suggest how to do this.